experiments/list_unsupported_build.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+
+echo >&2 "Project to build: ${project_dir}"
```

Comment:
We probably want the `${project_dir}` passed as a command line argument, rather than as a magic global environment variable.